features/install-wp-tests.feature

```diff
@@ -76,14 +76,14 @@ Feature: Scaffold install-wp-tests.sh tests
       """
 
     # Hardcoded for GHA, needs to be made more flexible for local setups.
-    When I run `mysql -u{DB_USER} -p{DB_PASSWORD} -h$MYSQL_HOST -P$MYSQL_TCP_PORT --protocol=tcp -e "SHOW DATABASES"`
+    When I run `MYSQL_PWD={DB_PASSWORD} mysql -u{DB_USER} -h{MYSQL_HOST} -P{MYSQL_PORT} --protocol=tcp -e "SHOW DATABASES"`
     Then the return code should be 0
     And STDOUT should contain:
       """
       wp_cli_test_scaffold
       """
 
-    When I run `WP_TESTS_DIR=/tmp/behat-wordpress-tests-lib phpunit -c {PLUGIN_DIR}/hello-world/phpunit.xml.dist`
+    When I run `WP_TESTS_DIR=/tmp/behat-wordpress-tests-lib /usr/local/bin/phpunit -c {PLUGIN_DIR}/hello-world/phpunit.xml.dist`
```

Comment:
really this should become part of the PATH that can be specified to the test suite, but this does a for-now fix and it will only ever be for GitHub Actions. 🤷